CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## 0.84.1
```

Comment:
It needs to be 0.84.0, and the build process will bump the patch itself to 0.84.1